index.js

```diff
@@ -69,24 +70,33 @@ function processFiles(fileNames, config) {
 	return Promise.all(fileNames.map(fileName => processFile(fileName, config)));
 }
 
-function processFile(fileName,
+async function processFile(fileName,
 		{ source, target, targetDir, fingerprint, assetManager, plugins }) {
 	let sourcePath = path.join(source, fileName);
 	let targetPath = path.join(target, fileName);
 
-	return readFile(sourcePath).
-		then(content => {
-			let type = determineFileType(sourcePath);
-			let plugin = type && plugins[type];
-			return plugin ? plugin(content) : content;
-		}).
-		then(content => {
-			let options = { targetDir };
-			if(fingerprint !== undefined) {
-				options.fingerprint = fingerprint;
-			}
-			return assetManager.writeFile(targetPath, content, options);
-		});
+	try {
+		var content = await readFile(sourcePath); // eslint-disable-line no-var
```

Comment:
```suggestion
		var content = await readFile(sourcePath, "utf8"); // eslint-disable-line no-var
```
no?

Reply:
No. Those could very well be binary files (images etc.). I don't think that this is a good idea.

Reply:
You're right, of course; I forgot about that.